pom.xml

```diff
@@ -760,10 +759,15 @@
 					<artifactId>maven-failsafe-plugin</artifactId>
 					<version>2.22.2</version>
 				</plugin>
+				<plugin>
+					<groupId>org.apache.maven.plugins</groupId>
+					<artifactId>maven-compiler-plugin</artifactId>
+					<version>3.8.1</version>
+				</plugin>
```

Comment:
Why do we need to add it as plugin now?

Reply:
I have updated this plugin to version 3.8.1 (from April 2019) when dealing with the compilation error mentioned above. Before, we used version 3.1 (from April 2013).